pkgs/by-name/tr/tracexec/package.nix

```
@@ -1,37 +1,60 @@
 {
   lib,
-  rustPlatform,
+  stdenv,
   fetchFromGitHub,
+  rustPlatform,
+  cargo-about,
 }:
-
-rustPlatform.buildRustPackage rec {
+let
   pname = "tracexec";
   version = "0.2.2";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
 
   src = fetchFromGitHub {
     owner = "kxxt";
-    repo = "tracexec";
+    repo = pname;
```

```suggestion
    repo = "tracexec";
```

https://github.com/nix-community/nixpkgs-lint/issues/21